### ISSUE TITLE
[1.10.2] Spice of Life Lunchbox Dupe

### ISSUE THREAD

**Issue by xSparky911x** [cpw/inventorysorter#34]
In a multiplayer server, when scrolling food from the Lunchbox the game does a temporary freeze and kicks you from the server but does not crash the game. When you join back in you have an extra food item in your inventory and still the same one in the Lunchbox. Not sure if this is caused by Spice of Life or Inventory Sorter based on the log. Does not seem to happen in single player

Forge: 2221
Inventory Sorter: 0.10.5+46

Log: http://pastebin.com/PhMyJS4L

**Comment by squeek502** [comment]
I can't reproduce this locally. Here's the [code in question](https://github.com/squeek502/SpiceOfLife/blob/1.9.4/java/squeek/spiceoflife/inventory/FoodContainerInventory.java#L30-L33), which should protect against this error:

```java
if (FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT)
{
	findMatchingClientItemStack();
}
```

My best guess is that it might be somehow related to #33?

**Comment by cpw** [comment]
Should be fixed with 1.12. I moved the processing to the server thread.
